lib/pacemaker/pcmk_sched_resource.c

```diff
@@ -621,12 +621,12 @@ cmp_resources(gconstpointer a, gconstpointer b, gpointer data)
     }
 
     // Calculate and log node weights
-    pcmk__native_merge_weights(convert_const_pointer(resource1),
-                               resource1->id, &r1_nodes, NULL, 1,
-                               pe_weights_forward | pe_weights_init);
-    pcmk__native_merge_weights(convert_const_pointer(resource2),
-                               resource2->id, &r2_nodes, NULL, 1,
-                               pe_weights_forward | pe_weights_init);
+    pcmk__add_colocated_node_scores(convert_const_pointer(resource1),
```

Comment:
Do we want to use the methods here too (future-proofing, etc.)?

Reply:
This one currently needs the direct call, but I think that should be avoidable by moving the group logic to the group method (in a later PR)